Lib/test/test_ast/test_ast.py

```diff
@@ -3203,26 +3203,29 @@ def wrap_for(self, for_statement):
         return ast.Module(body=[for_statement])
 
     def assert_ast(self, code, non_optimized_target, optimized_target):
+        def compare(left, right):
```

Comment:
Why not defining the function at the module-level, to not have to recreate the function at each assert_ast() call?